### DISCUSSION TITLE
Se alguém tiver com este erro -> Task :capacitor-camera:compileDebugJavaWithJavac FAILED

### DISCUSSION THREAD

**Discussion by GuilhermeAlamino** [ionic-team/capacitor#6619] [Q&A]
@Permission(strings = { Manifest.permission.CAMERA }, alias = CameraPlugin.CAMERA),
        // SDK VERSIONS 32 AND BELOW
        @Permission(
            strings = { Manifest.permission.READ_EXTERNAL_STORAGE, Manifest.permission.WRITE_EXTERNAL_STORAGE },
            alias = CameraPlugin.PHOTOS
        ),
        /*
        SDK VERSIONS 33 AND ABOVE
        This alias is a placeholder and the PHOTOS alias will be updated to use these permissions
        so that the end user does not need to explicitly use separate aliases depending
        on the SDK version.
         */
        @Permission(strings = { Manifest.permission.READ_MEDIA_IMAGES }, alias = CameraPlugin.MEDIA)

![image](https://github.com/ionic-team/capacitor/assets/75227213/c9f25fa8-0cf7-4504-af82-a765fac5550d)

        
aqui fala pra atualizar o sdk para 33 eu fiz isso e consegui, acaso não consiga tente contato comigo..

![image](https://github.com/ionic-team/capacitor/assets/75227213/03a9422a-9ef0-4d24-a70b-be7fa8a0b106)

**Comment by jcesarmobile** [comment]
version 5 of the camera plugin is supposed to be used in Capacitor 5, looks like you are using it in a previous version

**Reply by GuilhermeAlamino** [reply]
here it is working, i just needed to update the sdk to 33

**Reply by jcesarmobile** [reply]
To properly target SDK 33 you have to use capacitor 5, if not, other issues will arise.

**Reply by GuilhermeAlamino** [reply]
show thanks 🙌
